kvbc/include/Replica.h

```diff
@@ -186,6 +188,19 @@ class Replica : public IReplica,
   const std::shared_ptr<concord::secretsmanager::ISecretsManagerImpl> secretsManager_;
   std::unique_ptr<concord::kvbc::StReconfigurationHandler> stReconfigurationSM_;
   std::shared_ptr<cron::CronTableRegistry> cronTableRegistry_{std::make_shared<cron::CronTableRegistry>()};
+  concord::util::ThreadPool blocksIOWorkersPool_;
+
+ private:
+  static constexpr uint64_t MAX_VALUE_MICROSECONDS = 60ULL * 1000ULL * 1000ULL;  // 60 seconds
```

Comment:
the name is not self explanatory, max value for what?

Reply:
It's a convention we have in diagnostics (67 results on my search) when defining histograms.
This macro is inserted to DEFINE_SHARED_RECORDER